src/components/ReviewInfoText/index.tsx

```diff
@@ -61,7 +61,7 @@ export const ReviewInfoText = ({
 
     const transactionsToGo = safeNonceNumber - nonce
 
-    if (!shouldShowWarning) return null
+    if (!shouldShowWarning || isNaN(transactionsToGo)) return null
```

Comment:
Should we be showing anything when it is `NaN`?

Reply:
Why is a NaN in the first place?
Either `safeNonceNumber` or `nonce` isn't a number.

Reply:
This NaN occurs when waiting for the asynchronous call to fetch the recommended nonce (when calculating the `txParameters`).
After investigating, I think it is worth leaving this return condition as there will be cases where it can take a while to retrieve the `recomendedNonce` (and meanwhile we are showing the _"ReviewInfoText"_ message).

Reply:
Why not conditionally render this after the `recommendNonce` has finished loading?

Reply:
I don't think we should be blocking the UI...

Reply:
The fallback is the `<TransactionFees/>` component which seems to be a good fallback if we can't verify if the Tx is out of order

Reply:
I would render `<>` while it's loading. TransactionFees is quite large and cause flicker.

Reply:
I was just playing with it and it seems to flick less the way it is with `<TransactionFees/>`. So I would leave it as it is but let me know.

Reply:
Perfect, thanks for checking!